APKBUILD

```diff
@@ -1,21 +1,22 @@
 # Maintainer: Glider Labs <team@gliderlabs.com>
 
 pkgname="glibc"
-pkgver="2.23"
+pkgver="2.24"
+pkgrel="0"
 _pkgrel="0"
-pkgrel="3"
 pkgdesc="GNU C Library compatibility layer"
 arch="x86_64"
-url="https://github.com/sgerrand/alpine-pkg-glibc"
+url="https://github.com/dennybaa/alpine-pkg-glibc"
```

Comment:
definitely not required :+1:  It would've been nice if you had updated the glibc before but seems that you are just back and can pick what's need, specifically lib linking, glibc the newest version :)